lib/looker.ts

```diff
@@ -11,21 +14,30 @@ export async function getAWDashboardElement0(): Promise<IDashboardElement> {
   return elements[0];
 }
 
-export async function runEventCountQuery(filters: any): Promise<any>{
-  const element0 = await getAWDashboardElement0()
+export async function runEventCountQuery(filters: any, template: string): Promise<any>{
```

Comment:
Since the infobar query is not run against the event count look, let's change this function's name to `runQueryForTemplate`. I would also make the template the first parameter, as it's the most important thing.